lib/create_workout/set_timings.dart

```diff
@@ -122,25 +141,41 @@ class _SetTimingsState extends State<SetTimings> {
               mainAxisAlignment: MainAxisAlignment.center,
               children: [
                 const Icon(Icons.snooze),
-                IconButton(
-                  key: const Key('rest-decrement'),
-                  icon: const Icon(Icons.remove),
-                  onPressed: () => setState(() {
-                    final newValue = _restTime - 1;
-                    _restTime = newValue.clamp(1, 120);
-                    _restChanged = true;
-                  }),
-                ),
-                Text('Rest time: $_restTime seconds'),
-                IconButton(
-                  key: const Key('rest-increment'),
-                  icon: const Icon(Icons.add),
-                  onPressed: () => setState(() {
-                    final newValue = _restTime + 1;
-                    _restTime = newValue.clamp(1, 120);
-                    _restChanged = true;
-                  }),
-                ),
+                Column(children: [
+                  NumberPicker(
+                    value: _restTime,
+                    minValue: 1,
+                    maxValue: 50,
```

Comment:
```suggestion
                    maxValue: 120,
```
NumberPicker maxValue should match the max value of the element below